generic/tests/cfg/boot_vm_in_hugepage.cfg

```
@@ -20,13 +23,37 @@
         - 16M:
             no s390x
             expected_hugepage_size = 16384
+        - 32M:
+            only aarch64
+            kernel_hp_file = "/sys/kernel/mm/hugepages/hugepages-32768kB/nr_hugepages"
         - 512M:
             no s390x
-            expected_hugepage_size = 524288
+            variants:
+                - @default:
+                    expected_hugepage_size = 524288
+                - specify_hugepage_size:
```

```suggestion
                - specify_hp_file:
```
According to your mind.
I think the variant of `512M` represents the way to set the hugepage size: the `@default` means get it through the default file, such as `/proc/sys/vm/nr_hugepages`, and the another from the specify hp file,  such as `/sys/kernel/mm/hugepages/hugepages-xxxxx/nr_hugepages`
BTW, the `specify_hugepage_size` is a bit redundant since we know the huge page size is `512M` which declared the size is specified, so the name `variants` should not contain it.

---

Indeed, corrected :)